py/dml/ctree_test.py

```diff
@@ -9,7 +9,7 @@
 import math
 from pathlib import Path
 
-from dml import ctree, types, logging, messages, output, symtab
+from dml import codegen, ctree, types, logging, messages, output, symtab
```

Comment:
Oop. I imported this when I tried to construct `MethodFunc`s before quickly giving up. I'm removing this.